replit/crosis
# Move protocol to a separate package

Why
===
Moved the protocol (.proto) file to a separate repository that will be central repo for the message definitions. https://github.com/replit/protocol This allows us to have central place for all the consumers the .proto file.

Using an NPM package and listing the bindings as a peer dependency allows us to update the protocol without having to re-publish crosis.


What changed
============
- Deleted /protocol folder
- Add `@replit/protocol` as peer dependency and dev dependency
- Use `@replit/protocol` in code


Test plan
=========
- NPM link
- Import the library
- Make sure it works with peer dependency
